tests/scoreboard/MultiRun/run_test.pl

```diff
@@ -0,0 +1,75 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+  & eval 'exec perl -S $0 $argv:q'
+  if 0;
+
+use strict;
+use warnings;
+
+use FindBin;
+use constant autobuild_root => "$FindBin::Bin/../../../";
+$ENV{'PATH'} = "$ENV{'PATH'}:" . autobuild_root;
+use lib autobuild_root;
+chdir ($FindBin::Bin);
+
+use common::utility;
+use common::test_utils;
+
+use File::Path qw(make_path remove_tree);
```

Comment:
```suggestion
use File::Path qw(make_path);
```

Use `remove_tree` in `utility` that will print errors.

Reply:
rmtree() used in utility::remove_tree() seems to be the legacy interface of remove_tree() according to perldoc, and I don't know how perl interprets using remove_tree() to implement utility::remove_tree(), so I didn't change it. Also, in this test, remove_tree() is called as the last line for cleanup and its return is not considered as the test results. Maybe we should add this call at the beginning, too, to avoid running the test on the previously non-removed data.